experimenter/experimenter/glean/templates/glean/opt_out_alert.html

```diff
@@ -0,0 +1,31 @@
+{% if request.user.is_authenticated and not request.user.glean_prefs.alert_dismissed %}
+  <div id="glean-alert" class="alert alert-success bg-transparent text-success align-items-center">
+    <form method="post"
+          action="{% url 'glean-alert-dismissed' %}"
+          hx-post="{% url 'glean-alert-dismissed' %}"
+          hx-target="#glean-alert"
+          hx-swap="outerHTML">
+      <input
+        type="hidden"
+        name="alert_dismissed"
+        value="true"
+      />
+      {% csrf_token %}
+      <ul class="nav">
+        <li class="nav-item my-2 ms-2">
```

Comment:
I reused the nav and nav-item classes, because i wanted the list to default to horizontal and dynamically become vertical in small windows, but maybe this isn't the right way to do that.